go.mod

```diff
@@ -185,14 +171,19 @@ replace (
 	github.com/GoogleCloudPlatform/k8s-cloud-provider => github.com/GoogleCloudPlatform/k8s-cloud-provider v0.0.0-20190822182118-27a4ced34534
 	github.com/JeffAshton/win_pdh => github.com/JeffAshton/win_pdh v0.0.0-20161109143554-76bb4ee9f0ab
 	github.com/MakeNowJust/heredoc => github.com/MakeNowJust/heredoc v0.0.0-20170808103936-bb23615498cd
-	github.com/Microsoft/go-winio => github.com/Microsoft/go-winio v0.4.14
-	github.com/Microsoft/hcsshim => github.com/Microsoft/hcsshim v0.0.0-20190417211021-672e52e9209d
+	github.com/Microsoft/go-winio => github.com/Microsoft/go-winio v0.4.15-0.20190919025122-fc70bd9a86b5
+	github.com/Microsoft/hcsshim => github.com/Microsoft/hcsshim v0.8.7
```

Comment:
Based on this - we should still be using a tagged commit ID, not version. 0.8.7 wasn't tested with Kubernetes before it was tagged. If we want to use that as a testing point I'm ok with it, but I think we should tag the commit ID so the next person updating this doesn't assume there will be a relevant tagged version. 

https://github.com/microsoft/hcsshim/issues/642#issuecomment-571772207

Reply:
@PatrickLang yes, let's please use it as a testing point.